public/admin/config.yml

```
@@ -93,6 +93,23 @@ collections:
             required: false
             hint: Heading banner on top of Home page
             tagname: ""
+      - name: official_supported_apps
+        label: Official Supported Apps by Kemenkes
+        file: _content/official_supported_apps.json
```

```suggestion
      - name: officially_supported_apps
        label: Officially Supported Apps by Kemenkes
        file: _content/officially_supported_apps.json
```

Just a nitpick on its grammatical error. 😅 🙏

---

Done